src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicTests.cs

```diff
@@ -1052,34 +979,31 @@ public async Task BasicTest_WithReadsCompletedCheck()
                 iterations: 100,
                 serverFunction: async connection =>
                 {
-                    using QuicStream stream = await connection.AcceptStreamAsync();
-                    Assert.False(stream.ReadsCompleted);
+                    await using QuicStream stream = await connection.AcceptInboundStreamAsync();
+                    Assert.False(stream.ReadsClosed.IsCompleted);
 
                     byte[] buffer = new byte[s_data.Length];
                     int bytesRead = await ReadAll(stream, buffer);
 
-                    Assert.True(stream.ReadsCompleted);
+                    Assert.True(stream.ReadsClosed.IsCompleted);
```

Comment:
Should that be `IsCompletedSuccessfully`? in all places where we previously checked `ReadsCompleted` for `true`